CHANGELOG.adoc

```diff
@@ -4,6 +4,8 @@ All wesentlichen Änderungen für dieses Projekt werden hier dokumentiert.
 Das Format basiert auf link:https://keepachangelog.com/en/1.0.0[Keep a Changelog].
 
 ## [Unreleased]
+* API für Extensions, welche Nachrichten an einen Dienst schicken möchten.
```

Comment:
Das Ziel sollte in den Vordergrund gestellt werden:
bspw. - >Hilfsmethoden erstellen, um die Nutzung von ServiceNotifier Service in Extensions zu verinfachen.

Dazu auch die API-Klasse bitte nennen.